tock/utilization/analytics.py

```diff
@@ -82,29 +92,41 @@ def utilization_plot(data_frame):
 
     fig.update_layout(
         autosize=False,
-        width=1000,
-        height=700,
+        width=960,
+        height=720,
         xaxis=dict(autorange=True),
         yaxis=dict(autorange=True),
         title_text="Total Hours recorded vs. Time",
         hovermode="x",
+        legend=dict(
+            orientation="h",
+            yanchor="bottom",
+            y=1.02,
+            xanchor="right",
+            x=1,
+        ),
     )
 
     plot_div = plot(fig, output_type="div", include_plotlyjs=False)
     return plot_div
 
 
-def utilization_data(start_date, end_date):
+def utilization_data(start_date, end_date, org_id):
```

Comment:
This works but it feels like a great opportunity to re-use some of the other code we have a Tock -- there's a few places where we take URLParameters and return a filtered (or not) Queryset of Timecards and it would be awesome to have a single source for that work.

Could we expand/re-use maybe `api.views.get_timecards`? https://github.com/18F/tock/blob/64710825fadd6b8ade3c3de1290114815798d35a/tock/api/views.py#L192

Reply:
I hear you on code re-use, but in fact `get_timecards` is actually `get_timecardobjects` which fetches rows for the items on timecards rather than filtering the individual timecards which is what we're using here. 

I can definitely DRY up the first part of the queries in `utiltization_data` and `headcount_data`. Can we pair for more detail on what else you might have in mind?